sha1/src/lib.rs

```diff
@@ -26,6 +26,12 @@ use digest::const_oid::{AssociatedOid, ObjectIdentifier};
 #[cfg(feature = "zeroize")]
 use digest::zeroize::{Zeroize, ZeroizeOnDrop};
 
+#[cfg(feature = "std")]
+extern crate std;
+
+#[cfg(feature = "collision")]
+pub mod checked;
```

Comment:
I think it would make sense for this module name to match the feature name, e.g. either call them both `collision`, `checked`, or something else consistent

Reply:
yes makes sense, I don't like either name particularly, want to do a tiebreaker?

Reply:
`collision-detection` / `collision_detection` would be a lot clearer IMO, albeit verbose